### ISSUE TITLE
Jinn - Remote Assistance for React Native & Flutter

### ISSUE THREAD

**Issue by omranjamal** [DevArifHossain/saas-idea-validator#12]
# Jinn

## Description
A SaaS platform with an SDK that essentially provides AnyDesk or TeamViewer-like features but only for your own React Native or Flutter app. So when a customer calls, the sales and customer experience representative can send a popup and ask the user to press the consent button allowing the representative to temporarily take over their app.

## Features
- Remote Assistance Consent
- Virtual Cursor

## Future Features
- In-app chat
- In-app voice & video calls

## Use Cases
- Companies providing assistance to users for their app(s)

**Comment by ariflogs** [comment]
I used this feature in [crisp.chat](https://crisp.chat) where you can you(the site owner) can access your live user's website screen!!  Loved that feature.
Yeah, it will be great to have on mobile as well!

**Comment by omranjamal** [comment]
Are you talking about the co-browsing feature?

**Comment by ariflogs** [comment]
Nope.. so they have a chat plugin for websites but if a users need any help where you need to access the user's screen, you can do so by one of their features.
